setup.py

```diff
@@ -122,7 +122,6 @@ def build_extension(self, ext: CMakeExtension) -> None:
             f"-DDNNL_CPU_RUNTIME=OMP",
             f"-DNE_WITH_AVX2={'ON' if NE_WITH_AVX2 else 'OFF'}",
```

Comment:
some cmake flags don't make sense once once graph remove from ITREX.

Reply:
Yes, some cmake flag relate to cpp grape will be remove and relate to deprecate will be keep @intellinjun

Reply:
done

Reply:
Done? I think the deprecate executor will use `DNNL_CPU_RUNTIME` and `NE_WITH_AVX2`.

Reply:
You are right, deprecate executor will use NE_WITH_AVX2, no DNNL_CPU_RUNTIME